src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs

```diff
@@ -114,6 +114,8 @@ private static string CreateCorrelationContextHeader(RequestType requestType, Re
                 correlationContextKeyValues.Add(new KeyValuePair<string, string>(RequestTracingConstants.EnvironmentKey, RequestTracingConstants.DevEnvironmentValue));
             }
 
```

Comment:
If they never use feature management, the schema version is null and we shouldn't add this right?

Reply:
@zhenlan Do we want this for v1 schema or do we just want to know v2 schema usage?

Reply:
> If they never use feature management, the schema version is null and we shouldn't add this right?

The schema will be "1" by default, not null.

Reply:
If they're not using feature flags and we don't use the adapter and we send v1, that may be a bit misleading.

Reply:
Didn't think of that. I'll fix it, thanks!

Reply:
I thought we had a "UsesFeatureFlag" tracing, but we didn't. This will serve that purpose then :)

Reply:
The existing implementation relies on tracing feature flag usage by the ".appconfig.featureflag/" prefix in requests. But this would make it more explicit if we started logging v1 usage too.